client/store/auth.js

```diff
@@ -1,61 +1,90 @@
-import axios from 'axios'
-import history from '../history'
+import axios from 'axios';
+import history from '../history';
 
-const TOKEN = 'token'
+const TOKEN = 'token';
 
 /**
  * ACTION TYPES
  */
-const SET_AUTH = 'SET_AUTH'
+const SET_AUTH = 'SET_AUTH';
+const LOGOUT = 'LOGOUT';
 
 /**
  * ACTION CREATORS
  */
-const setAuth = auth => ({type: SET_AUTH, auth})
+const setAuth = (auth) => ({ type: SET_AUTH, auth });
 
 /**
  * THUNK CREATORS
  */
-export const me = () => async dispatch => {
-  const token = window.localStorage.getItem(TOKEN)
+export const me = () => async (dispatch) => {
+  const token = window.localStorage.getItem(TOKEN);
   if (token) {
     const res = await axios.get('/auth/me', {
       headers: {
-        authorization: token
-      }
-    })
-    return dispatch(setAuth(res.data))
+        authorization: token,
+      },
+    });
+    return dispatch(setAuth(res.data));
   }
-}
+};
 
-export const authenticate = (email, password, method) => async dispatch => {
+export const authenticate = (email, password, method) => async (dispatch) => {
```

Comment:
since you're being `authenticate` into `login` and `logout` you dont need this anymore

Reply:
Fixed!